lib/kennel/models/dashboard.rb

```diff
@@ -230,7 +230,7 @@ def validate_json(data)
 
         # Avoid diff from datadog presets sorting.
         presets = data[:template_variable_presets]
-        invalid! "template_variable_presets must be sorted by name" if presets && presets != presets.sort_by { |p| p[:name] }
+        invalid! :template_variable_presets_must_be_sorted, "template_variable_presets must be sorted by name" if presets && presets != presets.sort_by { |p| p[:name] }
```

Comment:
could shorten that a bit:
```suggestion
        invalid! :template_variable_presets_sorted, "template_variable_presets must be sorted by name" if presets && presets != presets.sort_by { |p| p[:name] }
```